actsims/signal.py

```diff
@@ -316,14 +313,9 @@ def load_alm_fg(self, set_idx, sim_idx):
         alm_fg90_150 = curvedsky.rand_alm_healpy(fg_power, seed = seed)#, lmax=lmax_sg)
         return alm_fg90_150
     
-    def get_template(self, patch,shape=None,wcs=None):
+    def get_template(self, patch, shape, wcs):
```

Comment:
Same comment as before, get_template can default to mapmaker footprint.

Reply:
But maybe no one ever needs it on the mapmaker footprint, and forcing people to specify the footprint is fine.